dawe_vocabs/settings.py

```diff
@@ -27,441 +27,1450 @@
+        uuid_namespace="substratum type",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-targeted-fauna-survey-types",
+        label="Targeted fauna survey types codes",
+        description="A collection of targeted fauna survey types and its " "codes.",
+        collection_uuid="70518904-e523-412e-8d3a-cdeae47a79e0",
+        uuid_namespace="targeted fauna survey types",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-targeted-survey-fauna-passive-equipments",
+        label="Targeted survey fauna passive equipments codes",
+        description="A collection of targeted survey fauna passive "
+        "equipments and its codes.",
+        collection_uuid="d1df6208-dec2-4f9c-91a8-2c00fc7835c1",
+        uuid_namespace="targeted survey fauna passive equipments",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-targeted-survey-fauna-vouchers",
+        label="Targeted survey fauna vouchers codes",
+        description="A collection of targeted survey fauna vouchers and its " "codes.",
+        collection_uuid="cb1fc752-939d-4311-ab19-7b2bc80cf3f3",
+        uuid_namespace="targeted survey fauna vouchers",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-targeted-survey-flora-growth-forms",
+        label="Targeted survey flora growth forms codes",
+        description="A collection of targeted survey flora growth forms and "
+        "its codes.",
+        collection_uuid="521dcaee-99f1-4df3-831c-6d0a27c4a530",
+        uuid_namespace="targeted survey flora growth forms",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-targeted-survey-flora-growth-stages",
+        label="Targeted survey flora growth stages codes",
+        description="A collection of targeted survey flora growth stages "
+        "and its codes.",
+        collection_uuid="096e018a-fb8f-4ba1-9fdc-302164e57682",
+        uuid_namespace="targeted survey flora growth stages",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-targeted-survey-general-survey-types",
+        label="Targeted survey general survey types codes",
+        description="A collection of targeted survey general survey types "
+        "and its codes.",
+        collection_uuid="7ca1606b-e8ed-4f7b-af2c-b3bdab30bae5",
+        uuid_namespace="targeted survey general survey types",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-taxa-types",
+        label="Taxa type codes",
+        description="A collection of taxa types and its codes.",
+        collection_uuid="7ea12fed-6b87-4c20-9ab4-600b32ce15ec",
+        uuid_namespace="taxa type",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-tecs",
+        label="Tecs codes",
```

Comment:
can we expand `tecs` to `threatened ecological communities`?

Reply:
Done.